src/components/Calculator.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import CalcButton from './Button';
+import CalcButton from './CalcButton';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Rather than disable the Linters, strive to fix them as this makes you more accustomed to the style guide of the tech.